app/src/main/java/com/sun/quickquotes/data/model/Quote.kt

```diff
@@ -23,9 +23,9 @@ data class Quote(
     @SerializedName("tag")
     val tag: String = "",
     @ColumnInfo(defaultValue = "0")
-    val isFavorite: Boolean = false,
+    var isFavorite: Boolean = false,
     @ColumnInfo(defaultValue = "-1")
-    val collectionId: Long = -1,
+    var collectionId: Long = -1,
```

Comment:
ở trên có defualtValue r dưới cx gán k bt có thừa k e

Reply:
không biết sao mà nếu cái model này được parse từ JSON, mà e muốn có 1 số parameter (không ở có trong JSON) có giá trị mặc định khi parse vào thì phải cho toàn bộ parameter có giá trị mặc định như thế a ạ. Còn cái @ColumnInfo(defaultValue = "-1") là của Room ạ